src/controllers/activityController.js

```diff
@@ -0,0 +1,30 @@
+const Activity = require("../models/Activity");
+
+// @desc    Endpoint for fetching all activities
+// @route   GET /api/v1/activity
+// @access  public
+const getAllActivities = async (req, res) => {
+  const response = await Activity.find();
+
+  return res.json({ count: response.length, activites: response });
+};
+
+// @desc    Endpoint for creating an activity
+// @route   POST /api/v1/activity/create
+// @access  public
+const saveActivity = async (req, res) => {
+  const { name, category, description, votes } = req.body;
+  if (!name || !category) {
+    res.status(400);
+    throw new Error("Activity name and category must be provided!");
+  }
```

Comment:
I was actually referring to the `errorHandler` and `notFound` functions you added in `src/middleware/errorHandler.js`. It doesn't look like those are used anywhere yet. I like the way you've explicitly written out the response status and messages in each case, and if it looks like there's opportunity for code reuse later, maybe we can write something similar to what's in that `errorHandler.js` file, but since it's not in use now, perhaps it could be removed?

Reply:
`errorHandler` and `notFound` are both middleware function that was implemented and utilized in this PR [#3](https://github.com/Code-the-Dream-School/ee-prac-team2-back/pull/3).

<img width="434" alt="image" src="https://github.com/Code-the-Dream-School/ee-prac-team2-back/assets/79583632/0ad4a19e-d740-4807-9671-f3065fcd8cd2">

Logic of the `errorHandler` middleware is quite implicit and mostly covers validation errors related to the database. As for `notFound` function, it is essentially just a global route to be triggered whenever a request is made to a route that doesn't exist (404 basically).

P.S. I should've properly described these changes in here [#3](https://github.com/Code-the-Dream-School/ee-prac-team2-back/pull/3). 😅